Podfile

```diff
@@ -6,7 +6,7 @@ target 'Signal' do
     pod 'AxolotlKit',                 git: 'https://github.com/WhisperSystems/SignalProtocolKit.git'
     #pod 'AxolotlKit',                 path: '../SignalProtocolKit'
     #pod 'SignalServiceKit',           git: 'https://github.com/WhisperSystems/SignalServiceKit.git'
-    pod 'SignalServiceKit',           git: 'https://github.com/WhisperSystems/SignalServiceKit.git', :branch => 'charlesmchen/webrtcByDefault'
+    pod 'SignalServiceKit',           git: 'https://github.com/WhisperSystems/SignalServiceKit.git', :branch => 'charlesmchen/callkitPrivacy'
```

Comment:
Can you be sure to change this back to master and `pod install` after merging the corresponding SSK Otherwise it's easy to end up referencing an orphaned SSK commit, which explodes when you run pod install.